lib/address.py

```diff
@@ -255,11 +256,15 @@ def to_ui_string(self, hex_only = False):
                     def lookup(x):
                         return OpCodes.reverseLookup.get(x, ('('+str(x)+')'))
                     if isinstance(op, tuple):
-                        ret += lookup(op[0]) + " " + op[1].decode('utf-8')
+                        try:
+                            opData = op[1].decode('utf-8')
+                        except:
```

Comment:
It's best to have the actual exception you expect

Reply:
I'll just change it to error='replace'